internal/gcs/container.go

```diff
@@ -30,26 +30,31 @@ var _ cow.Container = &Container{}
 
 // CreateContainer creates a container using ID `cid` and `cfg`. The request
 // will likely not be cancellable even if `ctx` becomes done.
-func (gc *GuestConnection) CreateContainer(ctx context.Context, cid string, cfg interface{}) (*Container, error) {
+func (gc *GuestConnection) CreateContainer(ctx context.Context, cid string, config interface{}) (_ *Container, err error) {
+	ctx, span := trace.StartSpan(ctx, "gcs::GuestConnection::CreateContainer")
+	defer span.End()
+	defer func() { oc.SetSpanStatus(span, err) }()
```

Comment:
Add cid as a span attribute?

Reply:
I can do that. I didnt add any context to these because the entry span from CD always sets the TaskID so it would be there on the trace. Up to you

Reply:
I think it's better to add identifying context to the spans. It shouldn't add much runtime cost, and could help with analysis (or if we ever call into here from a different caller).

Reply:
Done on all methods.